### ISSUE TITLE
Publishing of preprocessed ShapeNet models

### ISSUE THREAD

**Issue by angshine** [google-research/kubric#212]
Hi, thanks for the great work!

I want to render the ShapeNet models preprocessed by Kubric, but it seems that the ShapeNet models are not published yet. Is there any plan for releasing those models, like moving them to 'gs://kubric-public/'?

**Comment by MrXandbadas** [comment]
Hi there angshine!
Try looking at ShapeNet Taxonomy Viewer then downloading the models you want to use by clicking on the model and viewing its source! xD
[Taxonomy Viewer on shapenet.org](https://shapenet.org/taxonomy-viewer)

**Comment by angshine** [comment]
I think kubric comes with its own preprocessed ShapeNet models, as noted in the paper (w/ conversion scripts in`shapenet2kubric`). I'm wondering whether these preprocessed models would be released, or do I have to convert them myself.

**Comment by MrXandbadas** [comment]
It seems as though it's a JSON file in the code base... There are also some basic instructions inside shapenet.py under examples:

`# TODO: go to https://shapenet.org/ create an account and agree to the terms
#       then find the URL for the kubric preprocessed ShapeNet and put it here:
SHAPENET_PATH = "gs://KUBRIC_SHAPENET_PATH/ShapeNetCore.v2.json"

if SHAPENET_PATH == "gs://KUBRIC_SHAPENET_PATH/ShapeNetCore.v2.json":
  raise ValueError("Wrong ShapeNet path. Please visit https://shapenet.org/ "
                   "agree to terms and conditions, and find the correct path.")`

According to the comments left by the programmers it seems as though one has to make an account and locate the URL for the preprocessed ShapeNet objects.

To answer your question in regards to a free version of the object set I think we need to look at the Licensing for ShapeNet Itself:
[About us Page of ShapeNet.org](https://shapenet.org/about)
> Can I download the 3D models and annotations?
> For non-commercial research and/or educational purposes, we can provide download access to the processed mesh data and the associated annotation metadata through our site. To get access please sign up for a registered user account. 


N.B this would be a post for the Discussion Tab. This section is for problems and issues with the code or the like.

**Comment by MrXandbadas** [comment]
@angshine #185 
It should work out of the box?

**Comment by taiya** [comment]
TL;DR: it's already published and publicly available.

Sadly I couldn't paste the shapenet cloud bucket path directly, as you need to sign the agreement first.
Once you have it, you can browse the models one by one (no need to download the entire thing), or just
have kubric point to the asset source.
